src/main/java/com/leinardi/pycharm/mypy/mpapi/MypyRunner.java

```diff
@@ -272,19 +268,23 @@ public static String detectSystemMypyPath() {
         if (filesToScan.isEmpty()) {
             return Collections.emptyList();
         }
-        boolean daemon = false;
+        boolean daemon = true;
 
         GeneralCommandLine cmd = new GeneralCommandLine(mypyPath);
-        cmd.setCharset(Charset.forName("UTF-8"));
+        cmd.setCharset(UTF_8);
         if (daemon) {
             cmd.addParameter("run");
             cmd.addParameter("--");
-            cmd.addParameter("``--show-column-numbers");
-        } else {
-            cmd.addParameter("--show-column-numbers");
         }
+
+        cmd.addParameter("--show-column-numbers");
         cmd.addParameter("--follow-imports");
-        cmd.addParameter("silent");
+
+        if (daemon) {
+            cmd.addParameter("error");
+        } else {
+            cmd.addParameter("silent");
+        }
```

Comment:
For current mypy versions this should use `normal` which is the default when running plain `mypy`. I think the best would be to not set this parameter here at all to keep the default. That allows users to override it using the existing Arguments input field and set it to "error" or "silent" if they really want to (e.g. because they're using an old mypy version).

Reply:
I agree.